dd-java-agent/instrumentation/undertow-1.4/src/main/java/datadog/trace/instrumentation/undertow/UndertowInstrumentation.java

```diff
@@ -0,0 +1,99 @@
+package datadog.trace.instrumentation.undertow;
+
+import com.google.auto.service.AutoService;
+import datadog.trace.agent.tooling.Instrumenter;
+import datadog.trace.bootstrap.ContextStore;
+import datadog.trace.bootstrap.InstrumentationContext;
+import datadog.trace.bootstrap.instrumentation.api.AgentScope;
+import datadog.trace.bootstrap.instrumentation.java.concurrent.ExecutorInstrumentationUtils;
+import datadog.trace.bootstrap.instrumentation.java.concurrent.RunnableWrapper;
+import datadog.trace.bootstrap.instrumentation.java.concurrent.State;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+import java.util.Map;
+import java.util.concurrent.Executor;
+
+import static datadog.trace.agent.tooling.ClassLoaderMatcher.hasClassesNamed;
+import static datadog.trace.agent.tooling.bytebuddy.matcher.NameMatchers.named;
+import static datadog.trace.bootstrap.instrumentation.api.AgentTracer.activeScope;
+import static java.util.Collections.singletonMap;
+import static net.bytebuddy.matcher.ElementMatchers.isMethod;
+import static net.bytebuddy.matcher.ElementMatchers.takesArgument;
+
+@AutoService(Instrumenter.class)
+public final class UndertowInstrumentation extends Instrumenter.Tracing {
+
+  public UndertowInstrumentation() {
+    super("undertow", "wildfly");
```

Comment:
```suggestion
    super("undertow", "undertow-1.4");
```

If you change it to `undertow-2.0` change this to match

Reply:
Fixed!